ios/Plugin/FilePickerPlugin.swift

```diff
@@ -22,10 +23,20 @@ public class FilePickerPlugin: CAPPlugin {
 
         let multiple = call.getBool("multiple", false)
         let types = call.getArray("types", String.self) ?? []
-        let parsedTypes = parseTypesOption(types)
-        let documentTypes = parsedTypes.isEmpty ? ["public.data"] : parsedTypes
+        let fileExtensions = call.getArray("customExtensions", String.self) ?? []
+        if #available(iOS 14.0, *) {
+            let parsedTypes = newParseTypesOption(types)
+            let parsedExtensions = parseCustomExtensions(fileExtensions)
+            let concatenatedTypes = parsedTypes + parsedExtensions
+            let documentTypes = concatenatedTypes.isEmpty ? [.jpeg] : concatenatedTypes
```

Comment:
Why is the fallback changed to [.jpeg]?

Reply:
Well noted! The correct one is [.data], which has "public.data" as identifier